Crm/PersonAttributeForms.ascx.cs

```diff
@@ -845,6 +924,18 @@ private void ddlCountry_indexChanged( object sender, EventArgs e )
             upnlContent.Update();
         }
 
+        /// <summary>
+        /// Handles the SelectedIndexChanged event of the ddlFamilyMembers control.
+        /// </summary>
+        /// <param name="sender">The source of the event.</param>
+        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
+        protected void ddlFamilyMembers_SelectedIndexChanged( object sender, EventArgs e )
+        {
+            var qs = new Dictionary<string, string>();
+            qs.Add( "Person", ddlFamilyMembers.SelectedValueAsGuid().ToString() );
+            NavigateToCurrentPage( qs );
```

Comment:
This needs to handle other page parameters being kept, right now it discards them.